inscriptions/ISic000479.xml

```diff
@@ -186,23 +191,38 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/>memoriae
+                    <lb n="2"/> <name>Vegelae</name>
```

Comment:
You can lose the spaces at the start of the line before elements (we only need the spaces to separate out words in the line)